keyboards/mykeyclub/jris65/keymaps/via/keymap.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "dynamic_keymap": {
+            "layer_count": 5
+        },
+        "features": {
+            "via": true
+        }
+    },
+    "keyboard": "mykeyclub/jris65",
+    "keymap": "via",
+    "layout": "LAYOUT_all",
+    "layers": [
+                [
+                        "KC_ESC", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_BSLS", "KC_BSPC", "KC_PGUP"
+                        "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_PGDN"
+                        "KC_CAPS", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_BSLS", "KC_ENT", "KC_END"
+                        "KC_LSFT", "KC_BSLS", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_DOT", "KC_COMM", "KC_SLSH", "KC_RSFT", "KC_UP", "KC_DEL"
+                        "KC_LCTL", "KC_LGUI", "KC_LALT", "KC_SPC", "KC_RALT", "MO(1)", "KC_LEFT", "KC_DOWN", "KC_RGHT"
```

Comment:
Probably want to put back the layer with `QK_BOOT` here, as `MO(1)` goes out of bounds now.

Reply:
just what I was thinking 🤔

Reply:
I re-added the second layer to via